schemas/aws/policy-1.yml

```diff
@@ -18,6 +18,8 @@ properties:
     type: string
   description:
     type: string
+  mandatory:
+    type: boolean
```

Comment:
please add a `description`

Reply:
added in https://github.com/app-sre/qontract-schemas/pull/81/commits/e09e71dde11d20d358e759e5562a440e42638c84